.travis.yml

```diff
@@ -0,0 +1,13 @@
+---
+language: python
+python: "2.7"
+
+before_install:
+  - sudo apt-get update -qq
```

Comment:
Indeed! I added it because at first I thought `ansible-lint` would need `ansible` installed, but since it's not the case, let's delete it.